python/ccxt/bybit.py

```diff
@@ -585,10 +612,99 @@ def get_positions(self, symbol=None):
                         positions_to_return.append(result)
         return positions_to_return
 
-    def fetch_markets(self, params={}):
-        if self.options['adjustForTimeDifference']:
-            self.load_time_difference()
-        response = self.publicGetSymbols(params)
+    def fetch_spot_markets(self, params):
+        response = self.publicGetSpotV3PublicSymbols(params)
+        #
+        #     {
+        #         "ret_code":0,
+        #         "ret_msg":"",
+        #         "ext_code":null,
+        #         "ext_info":null,
+        #         "result":[
+        #             {
+        #                 "name":"BTCUSDT",
+        #                 "alias":"BTCUSDT",
+        #                 "baseCurrency":"BTC",
+        #                 "quoteCurrency":"USDT",
+        #                 "basePrecision":"0.000001",
+        #                 "quotePrecision":"0.00000001",
+        #                 "minTradeQuantity":"0.000158",
+        #                 "minTradeAmount":"10",
+        #                 "maxTradeQuantity":"4",
+        #                 "maxTradeAmount":"100000",
+        #                 "minPricePrecision":"0.01",
+        #                 "category":1,
+        #                 "showStatus":true
+        #             },
+        #         ]
+        #     }
+        markets_dict = self.safe_value(response, 'result', dict())
+        markets = self.safe_value(markets_dict, "list", list())
+        result = []
+        for i in range(0, len(markets)):
+            market = markets[i]
+            _id = self.safe_string(market, 'name')
+            base_id = self.safe_string(market, 'baseCoin')
+            quote_id = self.safe_string(market, 'quoteCoin')
+            base = self.safe_currency_code(base_id)
+            quote = self.safe_currency_code(quote_id)
+            symbol = base + '/' + quote
+            active = self.safe_value(market, 'showStatus')
+            quote_precision = self.safe_number(market, 'quotePrecision')
+            result.append({
+                'id': _id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'settle': None,
+                'baseId': base_id,
+                'quoteId': quote_id,
+                'settleId': None,
+                'type': 'spot',
+                'spot': True,
+                'margin': None,
+                'swap': False,
+                'future': False,
+                'option': False,
+                'active': active,
+                'contract': False,
+                'linear': None,
+                'inverse': None,
+                'taker': self.parse_number('0.001'),
```

Comment:
Not critic but we might change this to 0.001 as the parse number is float('0.001')